system_tests/nox.py

```diff
@@ -208,4 +214,37 @@ def session_compute_engine(session):
 
 def session_app_engine(session):
     session.virtualenv = False
-    session.run('pytest', 'app_engine/test_app_engine.py')
+
+    if SKIP_GAE_TEST_ENV in os.environ:
+        session.log('Skipping App Engine tests.')
+        return
+
+    # Unlike the default tests above, the App Engine system test require a
+    # 'real' gcloud sdk installation that is configured to deploy to an
+    # app engine project.
+    # Grab the project ID from the cloud sdk.
+    project_id = subprocess.check_output([
+        'gcloud', 'config', 'list', 'project', '--format',
+        'value(core.project)']).strip()
```

Comment:
An error will be raised and nox will mark the session as failed. If you want, I can catch it and use `session.error`.